ResearchKit/Common/CEVRKTheme.h

```diff
@@ -0,0 +1,55 @@
+//
+//  CEVRKTheme.h
+//  ResearchKit
+//
+//  Created by Eric Schramm on 7/10/19.
+//  Copyright © 2019 researchkit.org. All rights reserved.
+//
+
+@import UIKit;
+
+typedef NS_ENUM(NSInteger, CEVRKThemeType) {
+    CEVRKThemeTypeDefault,
+    CEVRKThemeTypeAllOfUs
+} ORK_ENUM_AVAILABLE;
+
+
+@class ORKBorderedButton;
+@class ORKContinueButton;
+
+@interface CEVRKTheme : NSObject
+
+- (nonnull instancetype)init NS_UNAVAILABLE;
++ (nonnull instancetype)initWithType:(CEVRKThemeType)type;
```

Comment:
`initWithType` looks a little odd in Swift code: `CEVRKTheme.initWith(.allOfUs)`. If you change this to the following:

    + (nonnull instancetype)themeWithType:(CEVRKThemeType)type;

Then in Swift you can write `CEVRKTheme(type: .allOfUs)` which looks better in my opinion (and also I think this is more idiomatic Objective-C naming)

Reply:
Oh, never mind, I see you already pushed a change that does exactly this.